script/build-flynn

```diff
@@ -0,0 +1,226 @@
+#!/bin/bash
+
+set -e
+
+ROOT="$(cd "$(dirname "$0")/.." && pwd)"
+source "${ROOT}/script/lib/ui.sh"
+
+usage() {
+  cat <<USAGE >&2
+usage: $0 [options]
+
+Build Flynn using the builder image.
+
+OPTIONS:
+  -h, --help              Show this message
+  -v, --verbose           Be verbose
+  -x, --version=VERSION   Explicit version to use [default: dev]
+  -t, --tuf-keys=KEYS     TUF root keys
+  --git-version           Generate the version using git status
+USAGE
+}
+
+main() {
+  local version="dev"
+  local verbose=false
+  local tuf_keys=""
+
+  while true; do
+    case "$1" in
+      -h | --help)
+        usage
+        exit 0
+        ;;
+      -v | --verbose)
+        verbose=true
+        shift
+        ;;
+      -x | --version)
+        if [[ -z "$2" ]]; then
+          fail "--version flag requires an argument"
+        fi
+        version="$2"
+        shift 2
+        ;;
+      -t | --tuf-keys)
+        if [[ -z "$2" ]]; then
+          fail "--tuf-keys flag requires an argument"
+        fi
+        tuf_keys="$2"
+        shift 2
+        ;;
+      --git-version)
+        version="$(git_version)"
+        shift
+        ;;
+      *)
+        break
+        ;;
+    esac
+  done
+
+  if [[ $# -ne 0 ]]; then
+    usage
+    exit 1
+  fi
+
+  # TODO: remove
+  rm -f "${ROOT}/cli/tuf.go"
```

Comment:
This is here so I can get things to build in CI where it still has files built from master which are interfering. I'll drop the WIP commit before merge, but I would like to see it pass in CI.